docs/roadmap.md

```diff
@@ -1,13 +1,15 @@
-# Kubernetes Roadmap
+# Kubernetes v1 
 
-Updated April 20, 2015
+Updated May 28, 2015
 
 This document is intended to capture the set of supported use cases, features,
 docs, and patterns that we feel are required to call Kubernetes “feature
-complete” for a 1.0 release candidate.  This list does not emphasize the bug
-fixes and stabilization that will be required to take it all the way to
-production ready.  This is a living document, and is certainly open for
-discussion.
+complete” for a 1.0 release candidate. 
+
+This list does not emphasize the bug fixes and stabilization that will be required to take it all the way to
+production ready. Please see the [Github issues] (https://github.com/GoogleCloudPlatform/kubernetes/issues) for a more detailed view. 
```

Comment:
`https://github.com/GoogleCloudPlatform/kubernetes/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.0` is the query for 1.0 issues.